partcad/src/partcad/lint/lint.py

```
@@ -0,0 +1,107 @@
+        result: LintingReport = await self.validate(ctx, package, target, lint_ctx)
+
+        await ctx.cache_lints.write_data_async(
+          hash,
+          {
+            cache_key: json.dumps(result.to_dict()).encode()
+          }
+        )
+
+        return result
+
+    async def lint_log_wrapper(self, ctx: Context, package: Project, target: str, lint_ctx: dict = {}) -> None:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)